core/src/main/java/org/elasticsearch/transport/RequestHandlerRegistry.java

```diff
@@ -77,6 +80,10 @@ public boolean isForceExecution() {
         return forceExecution;
     }
 
+    public boolean isCheckSizeLimit() {
```

Comment:
Well, what I like about the current name is that it captures the intention (skip the in-flight request limit check) but is not implementation specific (i.e. trip the circuit breaker). However, you seem to dislike it. What about a more specific name: `checkInflightRequestLimit`?